### PR TITLE
screen reader docs

### PR DESCRIPTION
**Pull request by danethurber**
screen reader docs

### CODE REVIEW

File: docs/src/pages/components/screenreaderonly.mdx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+name: Screen reader only
```
Comment (jaketrent) [inline docs/src/pages/components/screenreaderonly.mdx:2]: What is this `name` field used for?  Other .mdx files have had this lower-case and singleworded, afaik.

File: docs/src/pages/components/screenreaderonly.mdx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+route: /components/screenreaderonly
+---
+
+import ScreenReaderOnly from '@pluralsight/ps-design-system-screenreaderonly'
+
+import { Usage } from '../../components/component'
+
+# ScreenReaderOnly
```
Comment (jaketrent) [inline docs/src/pages/components/screenreaderonly.mdx:10]:
```suggestion
# Screen reader only
```

File: docs/src/pages/components/screenreaderonly.mdx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+}
+
+export default Example
+```
+## Usage
+
+<Usage
+  install="npm install @pluralsight/ps-design-system-theme"
+  import="import Theme from '@pluralsight/ps-design-system-theme'"
+  packageName="theme"
```
Comment (jaketrent) [inline docs/src/pages/components/screenreaderonly.mdx:29]:
```suggestion
  install="npm install @pluralsight/ps-design-system-screenreaderonly"
  import="import ScreenReaderOnly from '@pluralsight/ps-design-system-screenreaderonly'"
  packageName="screenreaderonly"
```

**Review by jaketrent — APPROVED** [review]

File: docs/src/pages/components/screenreaderonly.mdx (hunk 1 above)
Comment (danethurber) [reply docs/src/pages/components/screenreaderonly.mdx:2]: the idea is to use it to create the page title. i forgot to do that. i'll get a pr together for it

**Pull request merged**